src/headless.js

```diff
@@ -211,10 +211,9 @@ var XMLHttpRequest = function() {
       if (!this.async) {
         this.doSend();
       } else {
-        var that = this;
-        window.setTimeout(function() {
+        window.setTimeout(() => {
           that.doSend();
-          if (that.onload) that.onload();
+          if (this.onload) this.onload();
```

Comment:
Here also the change will break: `that` used to refer to the `XMLHttpRequest` object, but after this change, `this` will refer to the global scope (`window`). (also there's still a stray reference to `that` on line 215 above)

Reply:
Ditto, I think when we switch to arrow function we no longer need to capture the outer `this` in this way.

Reply:
Oh, indeed. Thanks, learned something new today!